packages/react-router-dom/modules/Prompt.js

```diff
@@ -1 +1 @@
-export default from 'react-router/Prompt'
+export { Prompt } from 'react-router'
```

Comment:
Don't you mean `export { Prompt as default } from 'react-router'`? Otherwise, people will have to `import { Prompt } from 'react-router-dom/Prompt'`